src/LogExpert/Extensions/MenuStripExtension.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace LogExpert.Extensions
+{
+    public class ExtendedMenuStripRenderer : ToolStripProfessionalRenderer
+    {
+        public ExtendedMenuStripRenderer() : base(new MenuSelectedColors()) { }
+    }
+
+    public class MenuSelectedColors : ProfessionalColorTable
+    {
+        public override Color ImageMarginGradientBegin
+        {
+            get { return LogExpert.Config.ColorMode.MenuBackgroundColor; }
```

Comment:
I would suggest using  

public override Color ImageMarginGradientBegin => LogExpert.Config.ColorMode.MenuBackgroundColor; 

in my opinion its more readable :)

Reply:
Done